### ISSUE TITLE
A well-defined route for SPA

### ISSUE THREAD

**Issue by martingg88** [balderdashy/sails#2148]
I've implemented the following router for SPA application to make all the route traffic to serve the index.html page but except for assets and /api. Actually, it does quite well. 
However it will redirect route traffic inherited from sails.js based application to this index.html as well. 
So far I found certain routes from socket like /__getcookie and  /firehose will be redirected to the following route. 

So what is the better method to prevent all of routes (inherited from sails based application) from redirected to following route. we really need  a nice way to handle this issue as we might not know what are the routes that are actually existed inherently in sails based application.

  '/*': {
	controller: 'angular',
	skipAssets: true,
	skipRegex: /^\/api\/.*$/ig
  }

**Comment by sailsbot** [comment]
Thanks for posting, @martingg88.  I'm a repo bot-- nice to meet you!

It has been 60 days since there have been any updates or new comments on this page.  If this issue has been resolved, feel free to disregard the rest of this message.  On the other hand, if you are still waiting on a patch, please:

  + review our [contribution guide](https://github.com/balderdashy/sails/blob/master/CONTRIBUTING.md) to make sure this submission meets our criteria (only _verified bugs_ with documented features, please;  no questions, commentary, or bug reports about undocumented features or unofficial plugins)
  + create a new issue with the latest information, including updated version details with error messages, failing tests, and a link back to [the original issue]().  This allows GitHub to automatically create a back-reference for future visitors arriving from search engines.

Thanks so much for your help!
